scripts/kind-e2e/lib_helm_deploy_subm.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# This should only be sourced
+if [ "${0##*/}" = "lib_helm_deploy_subm.sh" ]; then
+    echo "Don't run me, source me" >&2
+    exit 1
+fi
+
+function create_subm_vars() {
+    SUBMARINER_BROKER_NS=submariner-k8s-broker
+    SUBMARINER_PSK=$(cat /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 64 | head -n 1)
+    subm_ns=submariner
+}
+
+function install_helm() {
+    helm init --client-only
+    helm repo add submariner-latest https://submariner-io.github.io/submariner-charts/charts
+    pids=(-1 -1 -1)
+    logs=()
+    for i in 1 2 3; do
+        if kubectl --context=cluster${i} -n kube-system rollout status deploy/tiller-deploy > /dev/null 2>&1; then
+            echo Helm already installed on cluster${i}, skipping helm installation...
+        else
+            logs[$i]=$(mktemp)
+            echo Installing helm on cluster${i}, logging to ${logs[$i]}...
+            (
+            kubectl --context=cluster${i} -n kube-system create serviceaccount tiller
+            kubectl --context=cluster${i} create clusterrolebinding tiller --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+            helm --kube-context cluster${i} init --service-account tiller
+            kubectl --context=cluster${i} -n kube-system rollout status deploy/tiller-deploy
+            ) > ${logs[$i]} 2>&1 &
+            set pids[$i] = $!
+        fi
+    done
+    print_logs "${logs[@]}"
+}
+
+function deploytool_prereqs() {
+    install_helm
+}
+
+function preinstall_cleanup_subm() {
+    context=$1
+    delete_subm_pods $context submariner
+}
```

Comment:
This is duplicated in the operator deploy script. Is there a particular reason the base code doesn't directly call `delete_subm_pods $context $sub_ns`?

Reply:
Because the Helm/Operator deploys use different namespaces by default (submariner vs submariner-operator). This design hides that difference behind function calls with the same call signature (so we can do this dual Helm/Operator library import design).

Reply:
yeah the namespaces are different but the libraries already expose a `$sub_ns` var that is used elsewhere in the base code. So it seems like it could be used here as well. But not really a big deal either way.

Reply:
That makes sense. As @mangelajo raised below, the variables in e2e and the related libs need another set of refactoring. I raised #292 to track, and quoted/linked to this feedback. Imho this chunk of refactoring has been reviewed enough that it should go in as-is, handle #292 in a follow-up PR.

Reply:
sounds good.